src/kms/key-providers/secp256k1-provider.ts

```diff
@@ -59,37 +62,46 @@ export class Sec256k1Provider implements IKeyProvider {
    */
   async publicKey(keyId: KmsKeyId): Promise<string> {
     const privateKeyHex = await this.privateKey(keyId);
-    return this._ec.keyFromPrivate(privateKeyHex, 'hex').getPublic().encode('hex', false); // 04 + x + y (uncompressed key)
+    const publicKey = secp256k1.getPublicKey(privateKeyHex, false); // 04 + x + y (uncompressed key)
```

Comment:
rm `// 04 + x + y (uncompressed key)` comment. This comment is used to explain why we doing `slice(2);`, but not relevant here anymore.